app/js/components/LabOrdersList.jsx

```diff
@@ -40,7 +40,7 @@ const Cell = ({ columnName, value, handleCancel, cancelMsg, enableLabelPrinting,
       );
     }
     case 'NAME': {
-      const displayName = value.patient.display.split('-')[1].trim();
+      const displayName = ( value.patient.display && value.patient.display.indexOf('-') != -1) ? value.patient.display.split('-')[1].trim() : value.patient.person.display;
```

Comment:
Special treatment for people with hypens in their names?

Reply:
patient.display should look like this: 
`"display": "Y9C6ME - test, test 'test'"`
but, i just discovered today that if a patient has no ZL ID, then the display was null

Reply:
Wacky. Thanks

Reply:
Weird... I wonder if since you add person.display we should just use that all the time now? Is there any downside to that?  Seems like a better method. As Brandon mentioned,  with patients with hyphens in their names this current logic would screw things up (though of course that issue pre-dates this change).